### PR TITLE
Freeze Elasticsearch client version

### PR DESCRIPTION
**Pull request by Aschen**
## What does this PR do ?

Use the direct link to the latest version of the elasticsearch client supporting Open Search as well

### CODE REVIEW

**Review by xbill82 — APPROVED** [review]
Did you mean "freeze"?

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "lib": "lib"
   },
   "dependencies": {
-    "@elastic/elasticsearch": "7.15.0",
+    "@elastic/elasticsearch": "https://github.com/elastic/elasticsearch-js/archive/refs/tags/v7.13.0.tar.gz",
```
Comment (alexandrebouthinon) [inline package.json:42]:
```suggestion
    "@elastic/elasticsearch": "7.13.0",
```

**Review by alexandrebouthinon — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Aschen) [reply package.json:42]: The whole point of this PR is to stop using NPM version because it's upgraded by npm-check-update

**Review by alexandrebouthinon — APPROVED** [review]

**Comment by Aschen** [comment]
@xbill82
I mean always use this version because further versions are not compatible with OpenSearch. Until the SDK is forked or another solution we need to stick with this version

**Pull request merged**